### ISSUE TITLE
wxGTK/Wayland: Visual indicator of docking area with wrong position while dragging a wxAuiNotebookPage

### ISSUE THREAD

**Issue by taler21** [wxWidgets/wxWidgets#22576]
It's a Wayland specific problem as it works correctly with X11.
The issue can also be seen in the aui sample.

- When dragging the page to the left or top edge of the notebook, the indicator appears with an offset.

  wxGTK3 with Wayland:
  ![GTK3-Wayland_wxAuiNotebookPage-dragging-to-left](https://user-images.githubusercontent.com/99262969/176111337-ec151783-de9e-4b2c-a160-60844a8531e0.png)

  wxGTK3 with X11:
  ![GTK3-X11_wxAuiNotebookPage-dragging-to-left](https://user-images.githubusercontent.com/99262969/176111392-2461de29-a612-4741-add2-e0dda070c746.png)

- When dragging the page to the right or bottom edge of the notebook, the indicator appears as if the page was dragged to the opposite edge.

  wxGTK3 with Wayland:
  ![GTK3-Wayland_wxAuiNotebookPage-dragging-to-right](https://user-images.githubusercontent.com/99262969/176111554-b6ba82df-ee3b-4141-94b4-96ab1508af0d.png)

  wxGTK3 with X11:
  ![GTK3-X11_wxAuiNotebookPage-dragging-to-right](https://user-images.githubusercontent.com/99262969/176111585-1451a78b-291f-42ae-b877-a81bd8b498c8.png)

**Platform and version information**
 - wxWidgets version you use: 3.2.0-rc1
 - wxWidgets port you use: wxGTK
 - OS and its version: Ubuntu 22.04
 - For wxGTK only
   + GTK version: 3.24
   + Which GDK backend is used: Wayland
   + If relevant, i.e. for appearance-related problems: Adwaita-dark

**Comment by solemnwarning** [comment]
This may be fixed - I can't reproduce it with `master` on Debian 13 (GNOME or KDE).

**Comment by vadz** [comment]
Yes, thanks, this was probably fixed by AliKet's work in 29c8bfc249 (Use wxOverlay to show docking hint instead of transparent wxFrame, 2024-01-31). It seems work fine to me too now, so closing.

**Comment by taler21** [comment]
Indeed. I tested it and can confirm that this issue was fixed by 29c8bfc24925f1f0c67b7f9a8645a03b65e412a4 (`Use wxOverlay to show docking hint instead of transparent wxFrame, 2024-01-31`). Thanks!
